gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/InstantiatingHttpJsonChannelProvider.java

```diff
@@ -196,6 +196,7 @@ private HttpJsonTransportChannel createChannel() throws IOException, GeneralSecu
     HttpJsonClientInterceptor headerInterceptor =
         new HttpJsonHeaderInterceptor(headerProvider.getHeaders());
 
+    channel = new ManagedHttpJsonInterceptorChannel(channel, new HttpJsonLoggingInterceptor());
```

Comment:
nit: Can this work similar to grpcChannel where this either takes in a chain of `.intercept(INTERCEPTORS)` from the builder or a list of interceptors in the constructor? I don't think we need to do `new ManagedHttpJsonInterceptorChannel` twice.

I see this as package-private so we can do whatever is easier and modify it later as well.

Reply:
The current ManagedHttpJsonChannel.builder does not take `.intercept()`, `HttpJsonClientInterceptor` is a member added in `ManagedHttpJsonInterceptorChannel`. We can refactor later if we want to better parity between grpc and http setups.